src/Contrib/OtlpGrpc/Exporter.php

```diff
@@ -11,12 +11,15 @@
 use Opentelemetry\Proto\Collector\Trace\V1\ExportTraceServiceRequest;
 use Opentelemetry\Proto\Collector\Trace\V1\TraceServiceClient;
 use OpenTelemetry\SDK\EnvironmentVariablesTrait;
+use OpenTelemetry\SDK\Trace\Behavior\LoggerAwareTrait;
 use OpenTelemetry\SDK\Trace\Behavior\SpanExporterTrait;
 use OpenTelemetry\SDK\Trace\Behavior\UsesSpanConverterTrait;
 use OpenTelemetry\SDK\Trace\SpanExporterInterface;
+use Psr\Log\LoggerAwareInterface;
 
-class Exporter implements SpanExporterInterface
+class Exporter implements SpanExporterInterface, LoggerAwareInterface
 {
+    use LoggerAwareTrait;
```

Comment:
If you move the `LoggerAwareTrait` to the `SpanExporterTrait`, you don't have to add it here again.